model/project_ref.go

```diff
@@ -404,6 +404,47 @@ func (p *ProjectRef) GetPatchTriggerAlias(aliasName string) (patch.PatchTriggerD
 	return patch.PatchTriggerDefinition{}, false
 }
 
+func (p *ProjectRef) GetProjectParserMergedProjectRef(version string) *ProjectRef {
```

Comment:
Can we shorten this title somehow? Maybe "MergeWithParserProject"; we don't need to say ProjectRef again since this is a function of ProjectRef.

Reply:
Should probably add a short comment also to explain what happens with the input.